### DISCUSSION TITLE
Add collapsible Tab Groups to allow better tab management

### DISCUSSION THREAD

**Discussion by kwilcz** [zen-browser/desktop#1188] [Ideas]
Tab groups allow having lots of saved tabs in their specific groups, which allows us to return to the specific workflow easily.

**Comment by GitMurf** [comment]
FYI see here for a Discussion one of the maintainers opened up to follow this development: https://github.com/zen-browser/desktop/discussions/891

**Reply by kwilcz** [reply]
My bad, did not consider reading into Sidebery content.

**Reply by GitMurf** [reply]
No worries! Just trying to point people to a central location to talk about the related features so everyone can have a good discussion in one spot haha
